flutter_app/lib/main.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:flutter/material.dart';
+import 'command_constructor.dart';
 
 void main() {
   runApp(const MyApp());
+  testConstruction();
```

Comment:
Should move this call in the test file as well, although I'm not sure specifically how to run it...

Reply:
Setup testing with package:test and wrote tests. Can by run with `dart test` from flutter_app directory.